scripts/un/sdg/sdmx/cities.py

```diff
@@ -0,0 +1,90 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+'''Finds dcids for cities in input files.
+Produces:
+* preprocessed/cities.csv: dcid for each city name
+ 
+Note: For cities where the find entities API did not return a dcid,
+we tried manually searching for the dcid and filled these into the file.
+There are a few city names that are still missing - these are left blank.
+**This script ideally shouldn't need to be run again.**
+Usage: python3 cities.py <API_KEY>
+'''
+import csv
+import requests
+import os
+import sys
+
+BATCH = 1
+
+
+def get_cities(json, api_key):
+    '''Applies find entities API for given json.
+    Args:
+        json: Input json.
+        api_key: API key.
+    Returns:
+        API response.
+    '''
+    return requests.post(
+        'https://autopush.api.datacommons.org/v1/bulk/find/entities',
```

Comment:
Should this use api.datacommons.org so UN can run it?

Reply:
+1

Reply:
done